bower.json

```diff
@@ -17,6 +17,7 @@
     "font-roboto-local": "~1.0.1",
     "paper-scroll-header-panel": "~1.0.16",
     "app-layout": "~0.10.2",
-    "fecha": "~2.2.0"
+    "fecha": "~2.2.0",
+    "paper-range-slider": "git://github.com/IftachSadeh/paper-range-slider"
```

Comment:
Can you please specify a specific commit to import? We wouldn't want our future builds to break because the master branch got updated.

http://stackoverflow.com/questions/15774649/specifying-latest-revision-of-a-particular-branch-with-bower